va/meson.build

```diff
@@ -318,5 +318,9 @@ if WITH_WIN32
 endif
 
 foreach header : libva_headers_subproject
-  configure_file(output: fs.name(header), input: header, copy: true)
+  if meson.version().version_compare('>= 0.64')
+    fs.copyfile(header)
+  else
+    configure_file(output: fs.name(header), input: header, copy: true)
```

Comment:
Where I can find information on meson side that it got deprecated? I don't see that here: https://mesonbuild.com/Reference-manual_functions.html#configure_file